src/Solvers/fourier_tridiagonal_poisson_solver.jl

```diff
@@ -11,13 +13,14 @@ end
     i, j = @index(Global, NTuple)
     Nz = grid.Nz
 
-    D[i, j, 1] = -1/ΔzF(i, j, 1, grid) - ΔzC(i, j, 1, grid) * (λx[i] + λy[j])
+    # Using a homogeneous Neumann (zero Gradient) boundary condition:
+    D[i, j, 1] = - 1 / ΔzC(i, j, 2, grid) - ΔzF(i, j, 1, grid) * (λx[i] + λy[j])
 
     @unroll for k in 2:Nz-1
-        D[i, j, k] = - (1/ΔzF(i, j, k-1, grid) + 1/ΔzF(i, j, k, grid)) - ΔzC(i, j, k, grid) * (λx[i] + λy[j])
+        D[i, j, k] = - (1 / ΔzC(i, j, k+1, grid) + 1 / ΔzC(i, j, k, grid)) - ΔzF(i, j, k, grid) * (λx[i] + λy[j])
     end
 
-    D[i, j, Nz] = -1/ΔzF(i, j, Nz-1, grid) - ΔzC(i, j, Nz, grid) * (λx[i] + λy[j])
+    D[i, j, Nz] = -1 / ΔzC(i, j, Nz, grid) - ΔzF(i, j, Nz, grid) * (λx[i] + λy[j])
```

Comment:
We should probably switch to the newer/clearer notation Δzᵃᵃᶜ and Δzᵃᵃᶠ but could be part of a more comprehensive later PR since this is internal code.

Reply:
Yeah, I vote for a bug-fix release. You never know who else is using these features.

Reply:
Let's use Δzᵃᵃᶜ and Δzᵃᵃᶠ in this PR. I'll fix it up. Yes for bump.